### PR TITLE
Delete beta callout about "Transaction within CDC tables"

### LINKED ISSUE #1641: remove the beta warning for several features
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

- sql/commands/sql-commit.md
- sql/commands/sql-begin.md
- concepts/transactions.md

Although not frequently-used, "Transaction within CDC tables" is supposed to be stable enough. Please remove the callout from both the current version and the dev version.

- sql/data-types/data-type-rw_int256.md

`rw_int256` is also considered as a stable feature. Its requirement is solid, as there are a couple of crypto users using it in prod. 

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**
1. Delete the "Transaction within CDC tables" related beta callout from these files, both v1.5 and v1.6
    
    sql/commands/sql-commit.md
    sql/commands/sql-begin.md
    concepts/transactions.md

2. Delete "rw_int256" related beta callout from this file, both v1.5 and v1.6
    
     sql/data-types/data-type-rw_int256.md

3. BTW, delete descriptions from these files, both v1.5 and v1.6. 

    guides/ingest-from-citus-cdc.md
    guides/ingest-from-mysql-cdc.md
    guides/ingest-from-postgres-cdc.md

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1641

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Review by ShanlanLi — APPROVED** [review]
LGTM

File: docs/concepts/transactions.md [hunk 1]
```diff
@@ -33,9 +27,3 @@ Please note that data modifications are not allowed while a transaction is initi
 RisingWave supports transactions within a CDC table as a Beta feature. When you create a table to ingest CDC streams, you can enable this feature by setting `transactional` to `true` in the WITH clause of the `CREATE TABLE` statement. Note that this feature is only available if you are using the native [MySQL CDC](/guides/ingest-from-mysql-cdc.md), [PostgreSQL CDC](/guides/ingest-from-postgres-cdc.md), or [Citus CDC](/guides/ingest-from-citus-cdc.md) connectors.
```
Comment (hengm3467) [inline docs/concepts/transactions.md]:
```suggestion
When you create a table to ingest CDC streams, you can enable this feature by setting `transactional` to `true` in the WITH clause of the `CREATE TABLE` statement. Note that this feature is only available if you are using the native [MySQL CDC](/guides/ingest-from-mysql-cdc.md), [PostgreSQL CDC](/guides/ingest-from-postgres-cdc.md), or [Citus CDC](/guides/ingest-from-citus-cdc.md) connectors.
```

File: versioned_docs/version-1.5/concepts/transactions.md [hunk 2]
```diff
@@ -33,9 +27,3 @@ Please note that data modifications are not allowed while a transaction is initi
 RisingWave supports transactions within a CDC table as a Beta feature. When you create a table to ingest CDC streams, you can enable this feature by setting `transactional` to `true` in the WITH clause of the `CREATE TABLE` statement. Note that this feature is only available if you are using the native [MySQL CDC](/guides/ingest-from-mysql-cdc.md), [PostgreSQL CDC](/guides/ingest-from-postgres-cdc.md), or [Citus CDC](/guides/ingest-from-citus-cdc.md) connectors.
```
Comment (hengm3467) [inline versioned_docs/version-1.5/concepts/transactions.md]:
```suggestion
When you create a table to ingest CDC streams, you can enable this feature by setting `transactional` to `true` in the WITH clause of the `CREATE TABLE` statement. Note that this feature is only available if you are using the native [MySQL CDC](/guides/ingest-from-mysql-cdc.md), [PostgreSQL CDC](/guides/ingest-from-postgres-cdc.md), or [Citus CDC](/guides/ingest-from-citus-cdc.md) connectors.
```

**Review by hengm3467 — APPROVED** [review]
Otherwise LGTM. Might need to do a global search to see if there are some other occurrences of the similar sentences to the one I identified. Thanks.

File: versioned_docs/version-1.5/concepts/transactions.md (hunk 2 above)
Comment (WanYixian) [reply versioned_docs/version-1.5/concepts/transactions.md]: Thanks for pointing this out. 

Deleted the sentence above and found three more places that required deletion in the global search, pls take a look. Thanks!

guides/ingest-from-citus-cdc.md
guides/ingest-from-mysql-cdc.md
guides/ingest-from-postgres-cdc.md

File: docs/guides/ingest-from-citus-cdc.md [hunk 3]
```diff
@@ -12,10 +12,6 @@ RisingWave supports ingesting change data capture (CDC) data from Citus database
 
 Citus database is an extension to PostgreSQL that transforms PostgreSQL into a distributed database. For more details, see [Citus data](https://www.citusdata.com).
 
-:::note Beta Feature
```
Comment (neverchanje) [inline docs/guides/ingest-from-citus-cdc.md]: citus cdc is still in beta state. I didn't mention it in the original issue, as this feature is indeed never tested in prod.

File: docs/guides/ingest-from-mysql-cdc.md [hunk 4]
```diff
@@ -18,10 +18,6 @@ You can ingest CDC data from MySQL in two ways:
 
   With this connector, RisingWave can connect to MySQL databases directly to obtain data from the binlog without starting additional services.
 
-  :::note Beta feature
-  The built-in MySQL CDC connector in RisingWave is currently in Beta. Please contact us if you encounter any issues or have feedback.
-  :::
-
```
Comment (neverchanje) [inline docs/guides/ingest-from-mysql-cdc.md]: MySQL CDC is in beta and we plan to remove the mark by the end of this month, e.g. 1.6. We can remove the beta mark for dev, but not in 1.5.

File: versioned_docs/version-1.5/guides/ingest-from-postgres-cdc.md [hunk 5]
```diff
@@ -18,10 +18,6 @@ You can ingest CDC data from PostgreSQL into RisingWave in two ways:
 
   With this connector, RisingWave can connect to PostgreSQL databases directly to obtain data from the binlog without starting additional services.
 
-  :::note Beta feature
-  The built-in PostgreSQL CDC connector in RisingWave is currently in Beta. Please contact us if you encounter any issues or have feedback.
-  :::
-
```
Comment (neverchanje) [inline versioned_docs/version-1.5/guides/ingest-from-postgres-cdc.md]: Postgres CDC is unstable as well for now.

File: docs/guides/ingest-from-mysql-cdc.md (hunk 4 above)
Comment (WanYixian) [reply docs/guides/ingest-from-mysql-cdc.md]: Ok, I keep it in both 1.5 and 1.6 for now

File: docs/guides/ingest-from-citus-cdc.md (hunk 3 above)
Comment (WanYixian) [reply docs/guides/ingest-from-citus-cdc.md]: Thanks for clarifying. Add it back.

File: versioned_docs/version-1.5/guides/ingest-from-postgres-cdc.md (hunk 5 above)
Comment (WanYixian) [reply versioned_docs/version-1.5/guides/ingest-from-postgres-cdc.md]: Add it back to the doc.

**Review by neverchanje — APPROVED** [review]

**Pull request merged**